### PR TITLE
COMMERCE-12522 - Automate functional tests for COMMERCE-11798

### PR DESCRIPTION
**Pull request by jaydawu**
Test plan: https://liferay.atlassian.net/browse/COMMERCE-12482

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceReplacementProducts.testcase [hunk 1]
```diff
@@ -108,6 +108,126 @@ definition {
...
+			CommerceJSONSKUsAPI._addCommerceUOM(
+				basePrice = 30,
+				incrementalOrderQuantity = "1.5",
+				priority = 1,
+				productSKU = "MIN93015",
+				uomKey = "UOM2KEY",
+				uomName = "UOM2");
+		}
+
+		task ("And minium and multiple order quantities are set to '0.1' for ABS Sensor") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceReplacementProducts.testcase:135]:
```suggestion
		task ("And minimum and multiple order quantities are set to '0.1' for ABS Sensor") {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceReplacementProducts.testcase [hunk 2]
```diff
@@ -108,6 +108,126 @@ definition {
...
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/p/u-joint");
+		}
+
+		task ("Then assert that the 'Replacement Product' button redirects to ABS Sensor and that the UOM with highest priority is pre-selected(UOM2)") {
+			AssertElementPresent(
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceReplacementProducts.testcase:181]: Add a check for U-Joint's add to cart button disabled

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:forward

**Comment by jaydawu** [comment]
ci:forward

**Comment by jaydawu** [comment]
ci:forward failed, manually forward to BChan's repo, see https://github.com/brianchandotcom/liferay-portal/pull/142154

**Pull request closed without merging**